### PR TITLE
refactor utils

### PR DESCRIPTION
**Pull request by srinivasreddy**

### CODE REVIEW

**Comment by bep** [comment]
This looks fine; can you get the commit message in line with the guidelines.

**Comment by bep** [comment]
Please see https://github.com/spf13/hugo/blob/master/CONTRIBUTING.md

The problem isn't in the commit message's length or content, but its lack of capitalization and in your other PR the superfluous period at the end.

This may sound pedantic, but trying to be consequent here is the only really road to .... consequence.

**Comment by srinivasreddy** [comment]
@bep  Done.

**Comment by bep** [comment]
@srinivasreddy now I have a problem with the content of the commit message, it makes a mocking of the old code -- your original was better and more neutral.

The old code was fine. I'm not wasting too much more time on this one.

**Comment by srinivasreddy** [comment]
@bep  Apologies. None implied.

**Comment by srinivasreddy** [comment]
@bep  Please merge.

File: utils/utils.go [hunk 1]
```diff
@@ -20,35 +20,37 @@ import (
...
-			for _, message := range s {
-				if message != "" {
-					jww.CRITICAL.Println(message)
-				}
-			}
-		}
-		os.Exit(-1)
+        defer os.Exit(-1)
+
+	if len(s) == 0 {
```
Comment (bep) [inline utils/utils.go]: This doesn't look gofmt'ed

Comment (srinivasreddy) [inline utils/utils.go]: I am new to lang. Let me run that .

Comment (srinivasreddy) [inline utils/utils.go]: @bep  Done. Thanks.

**Comment by srinivasreddy** [comment]
ping @bep .

**Comment by bep** [comment]
Please don't ping me unless it's important. This isn't.

**Comment by spf13** [comment]
Merged as 6258d48

Thank you for your contribution @srinivasreddy

**Pull request closed without merging**